node/sc/multi_bridge_test.go

```diff
@@ -354,3 +356,592 @@ func TestMultiBridgeKLAYTransfer3(t *testing.T) {
+func TestMultiBridgeKLAYTransferParallel(t *testing.T) {
+	info := prepareMultiBridgeEventTest(t)
+	to := common.Address{100}
+	acc := info.accounts[0]
+
+	opts := &bind.TransactOpts{From: acc.From, Signer: acc.Signer, GasLimit: gasLimit}
+	tx, err := info.b.SetOperatorThreshold(opts, voteTypeValueTransfer, 2)
+	assert.NoError(t, err)
+	info.sim.Commit()
+	assert.Nil(t, bind.CheckWaitMined(info.sim, tx))
+
+	nonceOffset := uint64(17)
+	sentNonce := nonceOffset
+	transferAmount := uint64(100)
+	sentBlockNumber := uint64(100000)
+
+	go func() {
+		acc := info.accounts[0]
+		tx := SendHandleKLAYTransfer(info.b, acc, to, transferAmount, sentNonce, sentBlockNumber, t)
+		info.sim.Commit()
+		assert.NoError(t, bind.CheckWaitMined(info.sim, tx))
+
+		acc = info.accounts[1]
+		tx = SendHandleKLAYTransfer(info.b, acc, to, transferAmount, sentNonce, sentBlockNumber, t)
+		info.sim.Commit()
+		assert.NoError(t, bind.CheckWaitMined(info.sim, tx))
+	}()
+
+	go func() {
+		acc := info.accounts[2]
+		sentNonce := sentNonce + 20
+		tx := SendHandleKLAYTransfer(info.b, acc, to, transferAmount, sentNonce, sentBlockNumber, t)
+		info.sim.Commit()
+		assert.NoError(t, bind.CheckWaitMined(info.sim, tx))
+
+		acc = info.accounts[3]
+		tx = SendHandleKLAYTransfer(info.b, acc, to, transferAmount, sentNonce, sentBlockNumber, t)
+		info.sim.Commit()
+		assert.NoError(t, bind.CheckWaitMined(info.sim, tx))
+	}()
+
+	handleCounter := 0
+	for {
+		select {
+		case _ = <-info.handleCh:
+			handleCounter++
+			if handleCounter == 2 {
+				return
+			}
+		case err := <-info.handleSub.Err():
+			t.Fatal("Contract Event Loop Running Stop by sub.Err()", "err", err)
+		case <-time.After(timeOut):
+			// expected timeout
+			return
+		}
+	}
+}
+
+// TestMultiBridgeKLAYTransferMixConfig1 checks the following:
+// - set threshold to 2.
+// - the first tx is done.
+// - set threshold to 1.
+// - the second operator successfully handles the value transfer.
+func TestMultiBridgeKLAYTransferMixConfig1(t *testing.T) {
+	info := prepareMultiBridgeEventTest(t)
+	acc := info.accounts[0]
+	to := common.Address{100}
+
+	opts := &bind.TransactOpts{From: acc.From, Signer: acc.Signer, GasLimit: gasLimit}
+	tx, err := info.b.SetOperatorThreshold(opts, voteTypeValueTransfer, 2)
+	assert.NoError(t, err)
+	info.sim.Commit()
+	assert.NoError(t, bind.CheckWaitMined(info.sim, tx))
+
+	nonceOffset := uint64(17)
+	sentNonce := nonceOffset
+	transferAmount := uint64(100)
+	sentBlockNumber := uint64(100000)
+
+	tx = SendHandleKLAYTransfer(info.b, acc, to, transferAmount, sentNonce, sentBlockNumber, t)
+	info.sim.Commit()
+	assert.NoError(t, bind.CheckWaitMined(info.sim, tx))
+
+	opts = &bind.TransactOpts{From: acc.From, Signer: acc.Signer, GasLimit: gasLimit}
+	tx, err = info.b.SetOperatorThreshold(opts, voteTypeValueTransfer, 1)
+	assert.NoError(t, err)
+	info.sim.Commit()
+	assert.NoError(t, bind.CheckWaitMined(info.sim, tx))
+
+	acc = info.accounts[1]
```

Comment:
Previous acc (`acc := info.accounts[0]`) can also vote again to trigger execution.
Could you add the case? :)

Reply:
Good. Added.